bbq/vm/value.go

```diff
@@ -22,28 +22,58 @@ import (
 	"github.com/onflow/atree"
 
 	"github.com/onflow/cadence/bbq"
+	"github.com/onflow/cadence/common"
 	"github.com/onflow/cadence/interpreter"
 )
 
 type Value interface {
 	isValue()
-	StaticType(*Config) bbq.StaticType
+	StaticType(StaticTypeContext) bbq.StaticType
 	Transfer(
-		config *Config,
+		transferContext TransferContext,
 		address atree.Address,
 		remove bool,
 		storable atree.Storable,
 	) Value
 	String() string
 }
 
+type StaticTypeContext = interpreter.ValueStaticTypeContext
+
+type StorageContext interface {
```

Comment:
These (and below) contexts/interfaces are also removed in follow-up PRs, as the interpreter functions/values were re-used. So this is kind of temporary.